osquery/tables/utility/time.cpp

```diff
@@ -15,13 +15,25 @@
 namespace osquery {
 namespace tables {
 
+const char* WEEKDAYS[] = {"Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"};
+
 QueryData genTime(QueryContext& context) {
   Row r;
   time_t _time = time(0);
   struct tm* now = localtime(&_time);
+  char timestamp[21];
+  strftime(timestamp, sizeof(timestamp), "%FT%TZ", now);
```

Comment:
Will this be zulu time? I believe you'll need to convert local time to gmtime first.

Another thing I just remembered: in the schedule log there's a `calendarTime` field with the following format "Sat Jul  4 01:53:53 2015 UTC" and implementation: https://github.com/facebook/osquery/blob/master/osquery/core/system.cpp#L84

Maybe we should be consistent and call `timestamp` this calendar representation. The ISO representation is IMO "the best" but our existing format is (A) existing :/, and (B) the default from C++'s `asctime`.